By default the application will look for readme and help files in
current working directory. This patch changes the path for these
files to the correct location.

--- a/mainWindow.cpp
+++ b/mainWindow.cpp
@@ -923,7 +923,7 @@
 /* HELP MENU */
 void mainWindow::help()
 {
-    QDesktopServices::openUrl(QUrl("file:help/help.html" ));
+    QDesktopServices::openUrl(QUrl("file:/usr/share/doc/fxfloorboard-katana-mk2/help/help.html" ));
 }
 
 void mainWindow::whatsThis()
@@ -1097,7 +1097,7 @@
 
 void mainWindow::readMeDoc()
 {
-    QDesktopServices::openUrl(QUrl("file:readMe.txt"));
+    QDesktopServices::openUrl(QUrl("file:/usr/share/doc/fxfloorboard-katana-mk2/readme.txt"));
 }
 
 void mainWindow::replyFinished (QNetworkReply *reply)
